TouchPortalAPI/__init__.py

```diff
@@ -286,6 +324,33 @@ def stateUpdateMany(self, states:list):
         except TypeError:
             raise TypeError(f'StateUpdateMany() requires an iteratable, got {type(states)} instead.')
 
+    def showNotification(self, notificationId:str, title:str, msg:str, options:list):
+        if notificationId and title and msg and options and isinstance(options, list):
+            for option in options:
+                if not 'id' and 'title' in option.keys():
+                    raise TypeError("option require id and title keys")
+            self.send(
+                {
+                    "type": "showNotification",
+                    "notificationId": str(notificationId),
+                    "title": str(title),
+                    "msg": str(msg),
+                    "options": options
+                }
+            )
+
+    def connectorUpdate(self, connectorId:str, connectorValue:int):
+        if isinstance(connectorId, str):
+            if isinstance(connectorValue, int):
+                if 0 <= connectorValue <= 100:
+                    pass
+                else:
+                    raise TypeError(f"connectorValue needs to be between 0-100 not {connectorValue}")
+            else:
+                raise TypeError(f"connectorValue require a int not {type(connectorValue)}")
```

Comment:
does this do anything?

Reply:
Woops I think I forgot that part.... Thanks